templates/tools/dockerfile/test/cxx_sanitizers_jessie_x64/Dockerfile.template

```diff
@@ -15,7 +15,7 @@
   # limitations under the License.
 
   # This is the base Docker image we use for running tests on RBE
-  FROM gcr.io/cloud-marketplace/google/rbe-debian8@sha256:1ede2a929b44d629ec5abe86eee6d7ffea1d5a4d247489a8867d46cfde3e38bd
+  FROM gcr.io/cloud-marketplace/google/rbe-debian8@sha256:cda3a8608d0fc545dffc6c68f6cfab8eda280c7a1558bde0753ed2e8e3006224
```

Comment:
btw, this docker file is only used for run_tests.py sanitizers and it seems like we don't need them anymore
(and we are not even running them).
So at some point we're probably going to delete this dockerfile.

Reply:
It turned out that I changed the wrong file. As you said, it doesn't seem to be used. I can try to remove this file.